src/Resources/app/administration/test/e2e/cypress/integration/plugin-config.spec.js

```diff
@@ -0,0 +1,212 @@
+    function scrollToAndCheckLabel(selector, label) {
+        cy.contains(label.selector, label.text);
+        cy.get(selector).scrollIntoView().should("be.visible");
+    }
+
+    before(() => {
+        try {
+            return cy.createProductFixture();
+        } catch (_) {
+            // This allows this test file to be run multiple times without deleting the previous product.
+            // If the product is not created successfully the test will fail anyway
+        }
+    });
+
+    beforeEach(() => {
+        return cy.loginViaApi().then(() => {
+            return cy.setLocaleToEnGb();
+        });
+    });
+
+    it("should have a config button in the context menu", () => {
+        cy.visit("/admin#/sw/plugin/index/list");
+
+        // Open the Context Menu of the plugin
+        cy.get(".sw-card__content")
+            .contains("Label for the plugin PluginConfig")
+            .parents("tr")
+            .find(".sw-context-button__button")
+            .click();
+
+        // Click the context button
+        cy.get(".sw-context-menu")
+            .should("be.visible")
+            .contains("Config")
+            .should("be.visible")
+            .click();
+
+        cy.url().should("include", "/admin#/sw/plugin/settings/PluginConfig");
+    });
+
+    it("should be able to fill normal inputs", () => {
+        cy.visit("/admin#/sw/plugin/settings/PluginConfig");
+
+        const { elementSelector, label } = configElements.input;
+
+        scrollToAndCheckLabel(elementSelector, label);
+
+        cy.get(elementSelector).clearTypeAndCheck("test@shopware.com");
+    });
+
+    it("should be able to fill password inputs", () => {
+        cy.visit("/admin#/sw/plugin/settings/PluginConfig");
+
+        const { elementSelector, label } = configElements.input_password;
+
+        scrollToAndCheckLabel(elementSelector, label);
+
+        cy.get(elementSelector).clearTypeAndCheck("test@shopware.com");
+    });
+
+    it("should not be able to fill disabled inputs", () => {
+        cy.visit("/admin#/sw/plugin/settings/PluginConfig");
+
+        const { elementSelector, label } = configElements.input_disabled;
+
+        scrollToAndCheckLabel(elementSelector, label);
+
+        cy.get(elementSelector).should("be.disabled");
+    });
+
+    context("should be able to use single selects", () => {
+        const contexts = ["single_select_product", "single_select_mail"];
+
+        contexts.forEach((context) => {
+            it(context, () => {
+                const { options, elementSelector, label } = configElements[
+                    context
+                ];
+                if (!options || !options.length) {
+                    throw new Error(
+                        "Please provide non-empty options to a single select",
+                    );
+                }
+                scrollToAndCheckLabel(elementSelector, label);
+                for (const option of options) {
```

Comment:
probier hier auch mal nen Negativtest. ich zweifle immernoch dass for so gut funzt